crates/completion/src/config.rs

```diff
@@ -4,12 +4,15 @@
 //! module, and we use to statically check that we only produce snippet
 //! completions if we are allowed to.
 
+use assists::utils::MergeBehaviour;
+
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct CompletionConfig {
     pub enable_postfix_completions: bool,
     pub add_call_parenthesis: bool,
     pub add_call_argument_snippets: bool,
     pub snippet_cap: Option<SnippetCap>,
+    pub merge: Option<MergeBehaviour>,
```

Comment:
Can we not use the `MergeBehaviour` setting from the `InsertUseConfig` for this?

Reply:
Sort of related to https://github.com/rust-analyzer/rust-analyzer/pull/6553#discussion_r523456390

We sure can, but what should be the default? 
And would users be surprised to see two separate settings/behaviours for either of the cases?

Reply:
Ah right that's also part of the `assists` crate, so I guess in that case the config path for that should probably change if we move that stuff out of the `assists` crate.

I personally would expect import from completion and from the auto import assist to have the same behaviour in regards of merging, so I would expect only one setting. Otherwise you would get some import trees that are merged and some that aren't which I don't think anyone would like to have.

Reply:
Totally agree with a unified setting, that's what I've tried to achieve by reusing the same object.
Have I missed anything?

Reply:
Oh I'm sorry I misunderstood your intent, I kind of thought this would require to duplicate the setting in the config file in the end. Then forget what I was talking about that seems good 👍 
You probably want to copy the setting from the `InsertUseConfig` when the config gets loaded here though to have it in sync https://github.com/rust-analyzer/rust-analyzer/blob/master/crates/rust-analyzer/src/config.rs#L296-L306
As currently it's always gonna be `MergeBehaviour::Full` even if changed in the config file.

Reply:
Great catch, thank you.